recipe/meta.yaml

```diff
@@ -2,23 +2,25 @@
 {% set xorg_category = "lib" %}
 {% set name = "xorg-" ~ xorg_name %}
 {% set version = "1.1.3" %}
-{% set sha256 = "9c343225e7c3dc0904f2122b562278da5fed639b1b5e880d25111561bac5b731" %}
+{% set sha256 = "5bd9d4ed1ceaac9ea023d86bf1c1632cd3b172dce4a193a72a94e1d9df87a62e" %}
 {% set am_version = "1.15" %} # keep synchronized with build.sh
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.bz2
-  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.bz2
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://www.x.org/releases/individual/{{ xorg_category }}/{{ xorg_name }}-{{ version }}.tar.gz
```

Comment:
Why are we switching from the bz2 to the gzip here?

Reply:
In the latest X.org releases

https://www.x.org/releases/individual/lib/

They've changed from bz2 & gz to xz & gz. gz is the only format which is consistent.

Reply:
@pkgw

Reply:
We're ready when you're ready. @pkgw